src/test/ui/fn/issue-80179.stderr

```diff
@@ -11,10 +11,10 @@ error[E0121]: the placeholder `_` is not allowed within types on item signatures
   --> $DIR/issue-80179.rs:18:25
    |
 LL | fn returns_closure() -> _ {
-   |                         ^ not allowed in type signatures
-   |
-   = help: consider using an `Fn`, `FnMut`, or `FnOnce` trait bound
-   = note: for more information on `Fn` traits and closure types, see https://doc.rust-lang.org/book/ch13-01-closures.html
```

Comment:
I wonder if we should have kept the note with the link 🤔

Reply:
I can add it back

Reply:
It wasn't a request to do so, but I appreciate it :)